File: Leetcode/Easy/Arrays/signProductArray/signProductArray.js
var arraySign = function(nums) {
  let negative = 0;
  for (let num of nums){
    if (num < 0){
      negative++;
    }else if (num === 0){
      return 0;
    }
  }
  return (negative & 1) === 0 ? 1 : -1;
};

File: Leetcode/Easy/Arrays/signProductArray/signProductArray.py
def arraySign(self, nums: List[int]) -> int:
  negative = 0
  for num in nums:
    if num < 0:
      negative+=1
    elif num == 0:
      return 0
  if negative&1 == 0:
      return 1
  else:
      return -1